UART/UART.c

```diff
@@ -0,0 +1,120 @@
+/* This is the library of UART for Tiva C TM4C123GH6PM
+ *
+ * Made by : Quynh
+ *
+ */
+
+#include "UART.h"
+/*
+ * static variables used to setup UART according to the selected UART module
+ * default to UART0 (which is the computer UART)
+ */
+static uint32_t SYSCTL_PERIPH_GPIO    = SYSCTL_PERIPH_GPIOA;
+static uint32_t GPIO_UART_RX          = GPIO_PA0_U0RX;
+static uint32_t GPIO_UART_TX          = GPIO_PA1_U0TX;
+static uint32_t GPIO_UART_PORT        = GPIO_PORTA_BASE;
+static uint32_t GPIO_RX_PIN           = GPIO_PIN_0;
+static uint32_t GPIO_TX_PIN           = GPIO_PIN_1;
+static uint32_t UART_BASE             = UART0_BASE;
+static uint32_t UART_INT              = INT_UART0;
+
+/*Buffer and increment element declaration*/
+char UART_Buf_100_Char[100];
+uint8_t UART_Value_Count;
+
+
+/*Some UART function that can be used (must include the library uartstido.h and ustdilb.h)
+ *
+ * 1) UARTprintf(const char* pcString,.. ): Send a string through the UART
+ * 2) UARTgets(char *pcBuff, uint32_t ui32Len): Get the string sent from UART and save it into the buffer UART gets
+ * 3) UARTgetc(void): get a char from UART
+ * 4) UARTwrite(const char*pcBuf, uint32_t ui32Len) : Send a series of character (same with printf)
+ */
+
+
+/*
+ * Configure selected UART module
+ * @param <uint8> UART_module :
+ *          0: module uart 0 (PA0-RX, PA1-TX)
+ *          1: module uart 1 (PB0-RX, PB1-TX)
+ *          2: module uart 2 (PD6-RX, PD7-TX)
+ * @param <uint16_t> Baurate: Set the baurate for UART(usually 9600)
```

Comment:
Just a minor naming error in the comment session.
Suggested fix: `Baurate` -> `UART_baudrate`